xwiki-commons-core/xwiki-commons-xml/pom.xml

```diff
@@ -78,6 +78,11 @@
       <groupId>com.fasterxml.woodstox</groupId>
       <artifactId>woodstox-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.sf.carte</groupId>
+      <artifactId>xml-dtd</artifactId>
+      <version>3.0.0</version>
+    </dependency>
```

Comment:
@carlosame Hi. I didn't realized it back then when I merged your PR but we cannot do this since this artifact is currently NOT on Maven Central and all our deps inside xwiki-commons and xwiki-rendering must be on Maven Central since we deploy all these modules to Maven Central and they need to be self-standing (that's one rule of Maven Central).

Is there any possibility that you could deploy this artifact to Maven Central (through Sonatype's nexus repo as we do for ex)?

If not, I fear we'll have to rollback this PR and that would be a shame IMO.

Thanks a lot

Reply:
Hi,

Unfortunately my concerns explained in:

https://groups.google.com/g/css4j/c/op5jIoINb3M/m/IiiN-LfkDAAJ

do apply. There is no way that my artifacts are going into Central.